src/ReleaseHistory.md

```diff
@@ -1,5 +1,9 @@
 # SARIF Package Release History (SDK, Driver, Converters, and Multitool)
 
+* FEATURE: Multitool SARIF rewrite accepts `remove` parameter[#2160](https://github.com/microsoft/sarif-sdk/pull/2160)
+
+## **v2.3.8** [Sdk](https://www.nuget.org/packages/Sarif.Sdk/2.3.8) | [Driver](https://www.nuget.org/packages/Sarif.Driver/2.3.8) | [Converters](https://www.nuget.org/packages/Sarif.Converters/2.3.8) | [Multitool](https://www.nuget.org/packages/Sarif.Multitool/2.3.8) | [Multitool Library](https://www.nuget.org/packages/Sarif.Multitool.Library/2.3.8)
+
 * FEATURE: PACKAGE BREAKING: Upgrade from .NET Framework 4.5 to .NET Framework 4.5.2 [#2135](https://github.com/microsoft/sarif-sdk/pull/2135)
```

Comment:
>FEATURE [](start = 2, length = 7)

Eliminate whitespace on line 6? Go ahead and add the v2.3.9 banner at the top of the file?

Reply:
I was thinking about this and it may cause some confusion because it will show that we have already the version 2.3.9, which is not true. What i will do is change and create a ## Unreleased.
What do you think?

---
In reply to: [529763647](https://github.com/microsoft/sarif-sdk/pull/2160#discussion_r529763647) [](ancestors = 529763647)

Reply:
Sure, that's a fine idea. We just need to be disciplined about updating it when we actually release. 

---
In reply to: [529771234](https://github.com/microsoft/sarif-sdk/pull/2160#discussion_r529771234) [](ancestors = 529771234,529763647)